src/SUMMARY.md

```diff
@@ -32,6 +32,8 @@
 - [Control Flow Basics](control-flow-basics.md)
   - [Conditionals](control-flow-basics/conditionals.md)
   - [Loops](control-flow-basics/loops.md)
+    -[For](control-flow-basics/loops/for.md)
+    -[Loop](control-flow-basics/loops/loop.md)
```

Comment:
```suggestion
    - [`for`](control-flow-basics/loops/for.md)
    - [`loop`](control-flow-basics/loops/loop.md)
```
It looks like the missing whitespace was causing this not to appear -- good catch! Also, let's use backticks to format the keywords.